HopsanGUI/HcomHandler.h

```diff
@@ -102,6 +103,9 @@ class HcomHandler : public QObject
 
     void evaluateExpression(QString expr, VariableType desiredType=Undefined);
     double evaluateScalarExpression(QString expr, bool &rIsOK);
+    typedef double (*ScalarMathFunction_t)(double);
+    typedef QMap<QString, ScalarMathFunction_t> FuncMap_t;
+    void invokeSingleArgumentFunction(QString funcName, QString argStr, ScalarMathFunction_t f1);
```

Comment:
This functions is not defined, only declared, and do not seem to be used, obviously otherwise it would not compile. Assuming left-over, that should be removed.

Reply:
Approved, after removing or correcting this minor issu

Reply:
My bad, I removed the declaration.